.github/workflows/docker.yml

```diff
@@ -31,16 +31,17 @@ jobs:
         id: gitversion
         uses: gittools/actions/gitversion/execute@v0.9.7
       -
-        name: Publish release on GitHub
-        uses: softprops/action-gh-release@v1
+        name: Create tag for GitHub
+        uses: tvdias/github-tagger@v0.0.1
         with:
-          name: 'VulnerableApp-${{ steps.gitversion.outputs.semver }}'
-          tag_name: '${{ steps.gitversion.outputs.semver }}'
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          tag: '${{ steps.gitversion.outputs.semver }}'
       -
         name: Grant execute permission for gradlew
         run: chmod +x gradlew
       -
         name: Build with Gradle and Push
         run: |
           ./gradlew jib \
-          -Djib.to.tags='latest',$GITVERSION_SEMVER
\ No newline at end of file
+          -Djib.to.image=sasanlabs/owasp-vulnerableapp:unreleased \
+          -Djib.to.tags=$GITVERSION_SEMVER
```

Comment:
Here we can make the tag as:

```-Djib.to.tags=$GITVERSION_SEMVER,unreleased```

Reply:
It was like that before but Djib.to.tags [means additional tags](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#build-your-image).
When you build the image jib does not see these as to.tags so it will automatically append the 'latest' tag to the image then the additional tags. The avoid this behavior, either in the build.gradle or in Djib.to.image, use sasanlabs/owasp-vulnerableapp:unreleased. 

Found this issue with jib [here](https://github.com/GoogleContainerTools/jib/issues/2237). 

Using -Djib.to.tags=$GITVERSION_SEMVER,unreleased
would still append latest tag to the docker image.